src/components/FormField/FormField.module.scss

```
@@ -0,0 +1,61 @@
+.field {
+  position: relative;
+}
+
+
+
+.input,
```

❗️
Чёт много лишнего)
```suggestion
}

.input,
```